code/modules/surgery/organs/organ_external.dm

```diff
@@ -699,6 +709,9 @@ Note that amputating the affected organ does in fact remove the infection from t
 					throw_at(get_edge_target_turf(src,pick(alldirs)),rand(1,3),30)
 				dir = 2
 			return
+		else
```

Comment:
Alright, fair enough. I was also told that droplimb was infact properly stripped out, so this is probably safe enough.